### PR TITLE
Added serving ideas

### PR DESCRIPTION
**Pull request by ImanF01**

### CODE REVIEW

File: serving [hunk 1]
```diff
@@ -0,0 +1 @@
+put it on a plate
```
Comment (evasmere) [inline serving:1]: What dishes does guac work in?

File: serving [hunk 2]
```diff
@@ -1 +1,2 @@
 put it on a plate
+guac works in a plate, bowl, or cup
```
Comment (ImanF01) [inline serving:2]: Added more dishes

Comment (evasmere) [reply serving:2]: Great! I'll approve now

**Review by evasmere — APPROVED** [review]
All changes made, time to approve serving ideas

**Pull request merged**